types.d.ts

```diff
@@ -2595,8 +2595,8 @@ declare interface HashedModuleIdsPluginOptions {
 	hashFunction?: string;
 }
 declare class HotModuleReplacementPlugin {
-	constructor(options?: any);
-	options: any;
+	constructor(options?: {});
+	options: {};
 	multiStep: any;
```

Comment:
I've run `yarn special-lint-fix` to generate `types.d.ts`, but seems the types defined for `multiStep` and `fullBuildTimeout` are still `any`. Anything else I need to do?